src/plugins/console/public/application/components/help_panel.tsx

```diff
@@ -55,7 +55,7 @@ export function HelpPanel(props: Props) {
         </EuiTitle>
       </EuiFlyoutHeader>
       <EuiFlyoutBody>
-        <EuiText>
+        <EuiText size="s">
```

Comment:
we're generally just wrapping h1/h2 in euitext small to get consistent sizing - which screen shot is this?

Reply:
This is the Help Panel under Dev Tools:
Before:
<img width="390" alt="Help Before" src="https://github.com/user-attachments/assets/3451c9b3-3051-470f-8fc2-8b692687ace9">
After:
<img width="390" alt="Help Post" src="https://github.com/user-attachments/assets/bf6eb133-a116-4440-926c-d52ef7f6f91a">